tools/logger/logger.c

```diff
@@ -78,7 +79,7 @@ static void usage(void)
 
 static int snapshot(const char *name)
 {
-	const char *path = "/sys/kernel/debug/sof";
+	const char *path = SYS_DEBUG "/sof";
```

Comment:
it does not help readability, it was much cleaner when the whole path was humanly readable.

Reply:
Agreed but `sizeof("/sys/kernel/debug"/)` seemed ugly. I guess I can use a local variable there.